mix.exs

```diff
@@ -46,6 +46,7 @@ defmodule Uptime.MixProject do
       {:esbuild, "~> 0.8", runtime: Mix.env() == :dev},
       {:ex_machina, "~> 2.7", only: :test},
       {:excoveralls, "~> 0.18", only: :test},
+      {:file_system, "~> 1.0"},
```

Comment:
it looks like maybe this needs to be cleaned up, not sure if you actually ended up using it

Reply:
Shoot you're right, sorry about that